packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -68,30 +96,31 @@ const VisitNotesForm: React.FC<VisitNotesFormProps> = ({ patientUuid, closeWorks
       debounce((searchTerm) => {
         if (searchTerm) {
           const sub = fetchDiagnosisByName(searchTerm).subscribe(
-            (diagnoses: Array<Diagnosis>) => setSearchResults(diagnoses),
-            createErrorHandler(),
-            () => setIsSearching(false),
+            (diagnoses: Array<Diagnosis>) => setViewState((state) => ({ ...state, searchResults: diagnoses })),
+            () => createErrorHandler,
```

Comment:
You just mean `createErrorHandler`? Because `() => createErrorHandler` is a function that returns a function.